code/modules/clothing/neck/cloaks.dm

```diff
@@ -0,0 +1,47 @@
+//Cloaks. No, not THAT kind of cloak.
+
+/obj/item/clothing/neck/cloak
+	name = "grey cloak"
+	desc = "It's a cloak that can be worn around your neck in a pretty dull color."
+	icon_state = "cloak"
+	w_class = WEIGHT_CLASS_SMALL
+	body_parts_covered = UPPER_TORSO | ARMS
+
+/obj/item/clothing/neck/cloak/head_of_security
+	name = "head of security's cloak"
+	desc = "Worn by the leader of Brigston, ruling the station with an iron fist."
+	icon_state = "hoscloak"
+
+/obj/item/clothing/neck/cloak/quartermaster
+	name = "quartermaster's cloak"
+	desc = "Worn by the God-emperor of Cargonia, supplying the station with the necessary tools for survival."
+	icon_state = "qmcloak"
+
+/obj/item/clothing/neck/cloak/chief_medical_officer
+	name = "chief medical officer's cloak"
+	desc = "Worn by the leader of Medistan, the valiant men and women keeping pestilence at bay."
+	icon_state = "cmocloak"
+
+/obj/item/clothing/neck/cloak/chief_engineer
+	name = "chief engineer's cloak"
+	desc = "Worn by the leader of both Atmosia and Delamistan, wielder of unlimited power."
+	icon_state = "cecloak"
+
+/obj/item/clothing/neck/cloak/research_director
+	name = "research director's cloak"
+	desc = "Worn by the leader of Scientopia, the greatest thaumaturgist and researcher of rapid unexpected self disassembly."
+	icon_state = "rdcloak"
+
+/obj/item/clothing/neck/cloak/captain
+	name = "captain's cloak"
+	desc = "Worn by the supreme leader of Space Station 13."
```

Comment:
game name mentioned, roll credits.

Reply:
shhh it gets replaced with the station name on initialization